plugins/organic/organic.h

```diff
@@ -153,6 +153,8 @@ public slots:
 		MM_OPERATORS
 		Oscillator * oscLeft;
 		Oscillator * oscRight;
+		float * phaseOffsetLeft;
+		float * phaseOffsetRight;		
```

Comment:
I'd propose to rather use an `std::vector<float>` here. So something like this:
```C++
std::vector<float> phaseOffsetLeft;
std::vector<float> phaseOffsetRight;
```
This saves some allocations on the heap and also the array deletes starting on line 333 in the implementation file.